patroni/postgresql.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, config):
 
         self._version_file = os.path.join(self._data_dir, 'PG_VERSION')
         self._major_version = self.get_major_version()
+        self._synchronous_standby_names = None
```

Comment:
Isn't it already done here: https://github.com/zalando/patroni/pull/314/files#diff-57c53e988b775ee2902488fb33c1f433R279